test/scUSDC.unit.t.sol

```diff
@@ -0,0 +1,167 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.13;
+
+import "forge-std/console2.sol";
+import "forge-std/Test.sol";
+import {ERC20} from "solmate/tokens/ERC20.sol";
+import {WETH} from "solmate/tokens/WETH.sol";
+import {ERC4626} from "solmate/mixins/ERC4626.sol";
+import {FixedPointMathLib} from "solmate/utils/FixedPointMathLib.sol";
+import {IPool} from "aave-v3/interfaces/IPool.sol";
+import {IAToken} from "aave-v3/interfaces/IAToken.sol";
+import {IPoolDataProvider} from "aave-v3/interfaces/IPoolDataProvider.sol";
+
+import {scUSDC} from "../src/steth/scUSDC.sol";
+import {ISwapRouter} from "../src/interfaces/uniswap/ISwapRouter.sol";
+import {AggregatorV3Interface} from "../src/interfaces/chainlink/AggregatorV3Interface.sol";
+import {IVault} from "../src/interfaces/balancer/IVault.sol";
+
+import {MockERC4626} from "./mock/MockERC4626.sol";
```

Comment:
```suggestion
import {MockERC4626} from "solmate/test/utils/mocks/MockERC4626.sol";```

Reply:
addressed in [this](https://github.com/lindy-labs/sandclock-contracts/pull/56/commits/2a6e386c830954963fb98ddd98e1ab110a8b2b35) commit